atom/browser/api/lib/dialog.js

```diff
@@ -164,8 +162,9 @@ module.exports = {
 };
 
 // Mark standard asynchronous functions.
-ref1 = ['showMessageBox', 'showOpenDialog', 'showSaveDialog'];
+var ref1 = ['showMessageBox', 'showOpenDialog', 'showSaveDialog'];
+var j, len
 for (j = 0, len = ref1.length; j < len; j++) {
-  api = ref1[j];
+  var api = ref1[j];
```

Comment:
Thanks for catching these, I'll go back through and look more closely for these cases. 

We're taking the clean up bit by bit and will get to loops too :+1: